Registering a new user through Docker throws database errors

## What I did

I cloned the repo, copied and updated the `.env` file using SQLite for the database, built a container with `docker build -t lockbox .`, started a server with `docker run -it --rm -p 8080:80 lockbox`, and tried to register a user for testing.

## What I expected
I expected to be able to register a user and poke around the system a little bit.

## What actually happened
A 500 error was thrown because the `users` table doesn't exist yet.

```
returning exception no such table: users                                                                                          
Traceback (most recent call last):                                                                                                
  File "/usr/local/lib/python3.8/site-packages/aiosqlite/core.py", line 95, in run                                                
    result = function()                                                                                                           
sqlite3.OperationalError: no such table: users                                                                                    
[2021-01-14 15:23:38 +0000] [9] [ERROR] Exception in ASGI application     
Traceback (most recent call last):                                                                                       
  File "/usr/local/lib/python3.8/site-packages/aiosqlite/core.py", line 95, in run                                                
    result = function()                                                                                                           
sqlite3.OperationalError: no such table: users                                                                                    
[2021-01-14 15:23:38 +0000] [9] [ERROR] Exception in ASGI application                                                             
Traceback (most recent call last):                                                                                                
  File "/usr/local/lib/python3.8/site-packages/uvicorn/protocols/http/httptools_impl.py", line 390, in run_asgi                   
    result = await app(self.scope, self.receive, self.send)                                                                       
  File "/usr/local/lib/python3.8/site-packages/uvicorn/middleware/proxy_headers.py", line 45, in __call__                         
    return await self.app(scope, receive, send)                                                                                   
  File "/usr/local/lib/python3.8/site-packages/starlette/applications.py", line 111, in __call__                                  
    await self.middleware_stack(scope, receive, send)                                                                             
  File "/usr/local/lib/python3.8/site-packages/starlette/middleware/errors.py", line 181, in __call__                             
    raise exc from None                                                                                                           
  File "/usr/local/lib/python3.8/site-packages/starlette/middleware/errors.py", line 159, in __call__                             
    await self.app(scope, receive, _send)                                                                                         
  File "/usr/local/lib/python3.8/site-packages/starlette/middleware/sessions.py", line 75, in __call__                            
    await self.app(scope, receive, send_wrapper)                                                                                  
  File "/usr/local/lib/python3.8/site-packages/starlette/middleware/authentication.py", line 48, in __call__                      
    await self.app(scope, receive, send)                                                                                          
  File "/usr/local/lib/python3.8/site-packages/starlette/exceptions.py", line 82, in __call__                                     
    raise exc from None                                                                                                           
  File "/usr/local/lib/python3.8/site-packages/starlette/exceptions.py", line 71, in __call__                                     
    await self.app(scope, receive, sender)                                                                                        
  File "/usr/local/lib/python3.8/site-packages/starlette/routing.py", line 566, in __call__                                       
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.8/site-packages/starlette/routing.py", line 227, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.8/site-packages/starlette/routing.py", line 41, in app
    response = await func(request)
  File "/app/lockbox/routes/register.py", line 55, in real_register_endpoint
    if await user_already_exists(username):
  File "/app/lockbox/routes/register.py", line 30, in user_already_exists
    return (await database.fetch_one(query)) is not None
  File "/usr/local/lib/python3.8/site-packages/databases/core.py", line 146, in fetch_one
    return await connection.fetch_one(query, values)
  File "/usr/local/lib/python3.8/site-packages/databases/core.py", line 244, in fetch_one
    return await self._connection.fetch_one(built_query)
  File "/usr/local/lib/python3.8/site-packages/databases/backends/sqlite.py", line 104, in fetch_one
    async with self._connection.execute(query, args) as cursor:
  File "/usr/local/lib/python3.8/site-packages/aiosqlite/context.py", line 41, in __aenter__
    self._obj = await self._coro 
  File "/usr/local/lib/python3.8/site-packages/aiosqlite/core.py", line 160, in execute
    cursor = await self._execute(self._conn.execute, sql, parameters)
  File "/usr/local/lib/python3.8/site-packages/aiosqlite/core.py", line 109, in _execute
    return await future
  File "/usr/local/lib/python3.8/site-packages/aiosqlite/core.py", line 95, in run
    result = function()
sqlite3.OperationalError: no such table: users
```

## How to resolve it
The Dockerfile should run the migrations automatically when it's built so that when the server starts it's ready to go.

--- a/Dockerfile
+++ b/Dockerfile
@@ -7,4 +7,5 @@ COPY requirements.txt ./
 RUN pip install --no-cache-dir -r requirements.txt
 
 COPY . .
+RUN alembic upgrade head
 CMD [ "bash", "run_prod.sh", "--bind", "0.0.0.0:80" ]

